minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneAccumuloCluster.java

```diff
@@ -184,4 +188,12 @@ public ClusterUser getUser(int offset) {
     checkArgument(offset >= 0 && offset < users.size(), "Invalid offset, should be non-negative and less than " + users.size());
     return users.get(offset);
   }
+
+  @Override
+  public AccumuloConfiguration getSiteConfiguration() {
+    Configuration conf = new Configuration(false);
+    Path accumuloSite = new Path(serverAccumuloConfDir, "accumulo-site.xml");
```

Comment:
What happens if this file is unreadable (filesystem perms are off) or missing (misconfiguration from human error)? Should this fail hard with an uncheck Exception? Or just log WARN message?